1208-projects/user_edits_project.feature

```diff
@@ -0,0 +1,101 @@
+Feature: User edits Project
+
+    @xxx
+    Scenario: User opens the Project Wizard
+        # Denne er problematisk, alle brukere kan opprette eit prosjekt
+        Given User with access rights for Projects
+        When they clicks the Create New Project Button
+        Then they see the Project Wizard
+
+    @xxx
+    Scenario: User opens a Project in the Project Wizard
+        # Denne er problematisk, det stemmer i dette tilfellet, men det låser oss til å ikkje tillate brukere å opne eit prosjekt - med mindre dei gjer det fra landing page.
```

Comment:
Kan ikke det løses ved å legge til `And they are Manager of the Project` etter `Given`?

Reply:
Har rydda.